webapp/src/main/webapp/iaf/gui/js/controllers.js

```diff
@@ -817,7 +817,7 @@ angular.module('iaf.beheerconsole')
 		for(var i in $scope.configurations) {
 			var config = $scope.configurations[i];
 			$scope.isConfigStubbed[config.name] = config.stubbed;
-			$scope.isConfigReloading[config.name] = config.state != "STARTED"; //Assume reloading when not in state STARTED
+			$scope.isConfigReloading[config.name] = config.state == "STARTING" || config.state == "STOPPING"; //Assume reloading when not in state STARTED
```

Comment:
Ik zou na STOPPING een state STOPPED verwachten, of is STOPPING eigenlijk UNLOADING?

Reply:
Een configuratie is nooit gestopt. Stopping is inderdaad unloading